apps/web/src/components/forms/signin.tsx

```diff
@@ -73,6 +76,10 @@ export const SignInForm = ({ className }: SignInFormProps) => {
     }
   };
 
+  const onShowPassword = () => {
+    setShowPassword(!showPassword);
```

Comment:
Same same, also we can consider `toggleShowPassword`

Reply:
🫡

Reply:
Done and sent a fix below